### DISCUSSION TITLE
shm size clarification

### DISCUSSION THREAD

**Discussion by summitbri** [blakeblackshear/frigate#2057] [Ask A Question] [answered]
Hi, love the progress on this project and moving from the HA add-on to a docker container on a separate host.

I'm using one stream for object detection/rtmp roles and a higher resolution stream for recording.  Should I use the lower or higher resolution streams for the shm calculation?  Thanks!

**Comment by blakeblackshear** [comment, marked as answer]
The resolution from detect should be used.
